packages/apollo-tools/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.base",
   "compilerOptions": {
     "rootDir": "./src",
-    "outDir": "./lib"
+    "outDir": "./lib",
+    "esModuleInterop": true

```

Comment:
Any specific reason for this?

Reply:
Nope, must be an artifact of some exploration that I backtracked on. Removed!